doom-modeline-core.el

```diff
@@ -539,6 +539,12 @@ If nil, display only if the mode line is active."
   :type 'boolean
   :group 'doom-modeline)
 
+(defcustom doom-modeline-always-visible-segments nil
+  "A list of segments that should be visible even in
+inactive windows."
+  :type '(repeat string)
```

Comment:
Why is `string` type used here? May `symbol` be convient?
Anyway, what ever type, it's better provide an example in docstring and/or README.

Reply:
You are correct, thank you